src/PubkeyEncryptManager.php

```diff
@@ -243,4 +244,42 @@ public function deleteRoleKey(Role $role) {
       ->delete();
   }
 
+  /*
+   * Initialize Encryption Profiles upon module installation.
+   */
+  public function initializeEncryptionProfiles() {
+    // Load all Role keys.
+    $keys = \Drupal::service('key.repository')
+      ->getKeysByProvider('pubkey_encrypt');
```

Comment:
Can probably keep this to one line if it's <= 80 characters and there's only one method call (no chaining).